sklearn/multioutput.py

```diff
@@ -37,12 +39,89 @@ def _fit_estimator(estimator, X, y, sample_weight=None):
     return estimator
 
 
+def _partial_fit_estimator(estimator, X, y, classes=None, sample_weight=None,
+                           first_time=True):
+    if first_time:
+        estimator = clone(estimator)
+    else:
+        estimator = copy.deepcopy(estimator)
```

Comment:
it has been answered. it is to ensure fitting in different "process" on different estimators, and partial fit requires being called multiple times